java/src/main/java/org/apache/arrow/gandiva/evaluator/NativeBuilder.java

```diff
@@ -115,7 +115,7 @@ public String getByteCodeFilePath() {
    * Generates the LLVM module to evaluate the expressions with
    * custom configuration.
    *
-   * @param schemaBuf   The schema serialized as a protobuf. See Types.proto
+   * @param schemaBuf   The schema serialized as a protobuf. See ExpressionRegistry.proto
```

Comment:
cut-paster error ?

Reply:
looks like intellij replaced all occurences of types with later on refactoring :)